#!/usr/bin/env python

PACKAGE = "singaboat_vrx"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("thrust_limit", double_t, 0, "Maximum permitted thrust command for ASV", 1.0, 0, 1.0)
gen.add("thrust_step", double_t, 0, "Stepping amount to alter thrust command for ASV", 0.2, 0, 1.0)

gen.add("debug", bool_t, 0, "Flag to enable/disable debug messages", True)

exit(gen.generate(PACKAGE, "singaboat_vrx", "Teleoperation"))
